pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -118,6 +118,13 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
+.PHONY: test-coverage
+test-coverage: ## Run unit tests creating the output to report coverage
+	- rm -rf *.out  # Remove all coverage files if exists
+	go test -race -failfast -tags=integration -coverprofile=cover.out.tmp
+	cat cover.out.tmp | grep -v "zz_generated.deepcopy.go" > cover.out
+	tool cover -func cover.out
+	
```

Comment:
@camilamacedo86 Could you please take a look.
Does removing the file from the cover file solved the issue here?
Is this approach used reasonable? Please inform.

Reply:
I have a few thoughts on this:

- why is the `rm` prefixed with "- "? I've never seen that before. Is that some kind of makefile syntax? If so, then what does it do?
- I think the the `tool cover` line needs to be prefixed with `go `
- I think another line can be added at the end to delete the tmp file, i.e. `rm cover.out.tmp`
- In general, not sure if this is the right approach. It feels more like a workaround, i.e. it still won't fix the output displayed in the `make test` output